src/Microsoft.AspNetCore.SignalR.Core/IHubContext.cs

```diff
@@ -3,10 +3,19 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A context abstraction for a hub.
+    /// </summary>
     public interface IHubContext<THub> where THub : Hub
     {
+        /// <summary>
+        /// Gets the clients for the hub.
```

Comment:
Gets an object that can be used to invoke methods on clients connected to the hub.